distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -2374,9 +2375,11 @@ type Table
 
              fill_nothing = table.fill_nothing ["col0", "col1"] 20.5
     @columns Widget_Helpers.make_column_name_vector_selector
-    fill_nothing : Vector (Integer | Text | Regex) | Text | Integer | Regex -> Column | Any -> Table
-    fill_nothing self columns default =
-        transformer col = col.fill_nothing default
+    @default Widget_Helpers.make_column_ref_by_name_selector
+    fill_nothing : Vector (Integer | Text | Regex) | Text | Integer | Regex -> Column | Column_Ref | Any -> Table
```

Comment:
I assume that the reason we don't define a conversion `Column` -> `Column_Ref` is because `Column_Ref` is meant to only mean a reference to a column within the table?

Reply:
1. First of all, I don't think this conversion would serve any purpose. Sometimes we want to pass a `Column` instance and sometimes we want to refer to a column in a specific table by name, without confusing it with a plain text value - these are different use cases, although indeed slightly similar.
2. In general, we cannot define a conversion from `Column` to `Column_Ref`, because `Column_Ref` has to be interpreted in the context of a specific `Table`, but a `Column` instance may come from various tables - so we'd have to guarantee that the conversion only happens for columns coming from the expected table. And in in-memory we don't even have a notion of what table a column 'comes from'.

a.d. 2 Example:
```
t1 = Table.new [["X", [1,2,3]]]
t2 = Table.new [["Y", ['a', 'b', 'c']]]

r = t1.set_OR_SOME_OTHER_OP (t2.at "Y")
```

Assuming we had a conversion `Column -> Column_Ref` to fire in `set_OR_SOME_OTHER_OP`, we can see that it cannot be performed - there is no column `Y` in `t1` that we want to refer to, and if it were, it could be a different column than the one we mean here.